src/main/java/org/jetbrains/bsp/bazel/server/bsp/resolvers/actiongraph/ActionGraphV1Parser.java

```diff
@@ -1,45 +1,28 @@
-package org.jetbrains.bsp.bazel.server.bsp.resolvers;
+package org.jetbrains.bsp.bazel.server.bsp.resolvers.actiongraph;
 
 import com.google.common.collect.Lists;
 import com.google.devtools.build.lib.analysis.AnalysisProtos;
-import java.util.ArrayDeque;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Queue;
-import java.util.Set;
+import java.util.*;
```

Comment:
Where will this be resolved? What did you guys decide?

Reply:
#45 is going to fix that